src/ansys/aedt/core/maxwell.py

```diff
@@ -2961,6 +2961,132 @@ def assign_zero_tangential_h_field(self, assignment, boundary=None):
             return bound
         return False
 
+    @pyaedt_function_handler()
+    def assign_resistive_sheet(
+        self,
+        assignment,
+        resistance="1ohm",
+        name=None,
+        non_linear=False,
+        anode_a="300000000",
+        anode_b="5",
+        anode_c="110000000000000",
+        anode_d="2",
+        cathode_a="300000000",
+        cathode_b="10",
+        cathode_c="110000000000000",
+        cathode_d="2",
+    ):
+        """Assign a resistive sheet boundary between two conductors.
+
+        Available for Maxwell 3D Magnetostatic, Eddy Current and Transient designs.
+        For 3D Magnetostatic designs, the user can specify the nonlinear anode and cathode coefficients.
+        To understand the nonlinear relationship used by AEDT between the conductivity and current density,
+        please refer to Maxwell Help guide.
+
+        Parameters
+        ----------
+        assignment : list of int or :class:`ansys.aedt.core.modeler.cad.object_3d.Object3d`
```

Comment:
```suggestion
        assignment : list of int, str or :class:`ansys.aedt.core.modeler.cad.object_3d.Object3d`
```
Just wondering, can't assignment be an `str` ?

Reply:
@SMoraisAnsys The native mathod doesn't accept the names because face objects don't have names but only ids.